src/core/Akka.Remote/RemoteActorRefProvider.cs

```diff
@@ -189,7 +189,7 @@ public ActorPath RootPath
         public Deployer Deployer { get; protected set; }
 
         /// <inheritdoc/>
-        public Address DefaultAddress { get { return Transport.DefaultAddress; } }
+        public Address DefaultAddress { get { return Transport?.DefaultAddress; } }
```

Comment:
Make sure we return null instead of throwing an NRE if `Transport` is still null.